modules/nf-core/vt/normalize/main.nf

```diff
@@ -0,0 +1,72 @@
+
+
+process VT_NORMALIZE {
+    tag "$meta.id"
+    label 'process_low'
+
+    // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+    conda "bioconda::vt=2015.11.10"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/vt:2015.11.10--h5ef6573_4':
+        'biocontainers/vt:2015.11.10--h5ef6573_4' }"
+
+    input:
+    tuple val(meta) , path(vcf), path(tbi), path(intervals)
+    tuple val(meta2), path(fasta)
+    tuple val(meta3), path(fai)
+
+    output:
+    tuple val(meta), path("*.vcf.gz")       , emit: vcf
+    tuple val(meta), path("${fasta}.fai")   , emit: fai, optional: true
```

Comment:
Feels a bit odd to export this, we don't normally worry about intermediate files that are produced by the tool.